hydrator-common/src/main/java/co/cask/hydrator/common/SchemaValidator.java

```diff
@@ -116,6 +120,25 @@ public static void validateFieldsArePresentInSchema(Schema schema, String... req
     }
   }
 
+  /**
+   * Checks that the given schema is not null
+   *
+   * @param schema the schema to be checked
+   * @param name the name of the schema
+   * @return true if the schema is not null else false
+   */
+  public static boolean validateSchemaIsNotNull(Schema schema, String name) {
```

Comment:
should be annotated as nullable

Reply:
methods that return a boolean should start with 'is' or 'should' or something like that. Also, these log messages are very specific to FLL. 'canRecordLineage()' would be a better name.